commercial/app/controllers/Multi.scala

```diff
@@ -128,7 +128,7 @@ class Multi(bestsellersAgent: BestsellersAgent,
         Cached(componentMaxAge) {
           JsonComponent(JsArray((offerTypes zip contents).map { case (contentType, content) => Json.obj(
               "type" -> contentType,
-              "value" -> Json.toJson(content)(Merchandise.writes)
+              "value" -> Json.toJson(content)(Merchandise.merchandiseWrites)
```

Comment:
Unfortunately, despite using a `sealed trait` - the compiler doesn't seem able to infer that there is a Json serializer for all implementations of `Merchandise`, thus it complains `No Json serializer found for type model.commercial.Merchandise.`.

It's a bit annoying since the whole point of sealed traits is to allow this kind of compilation inference :/